ISISScientificComputing/autoreduce
# Queue Processor - small logging improvements

### Summary of work
- Now properly logs the `reduction_log` on error cases so that "Show reduction log" on the web app shows the error
- Logs the hostname of the worker, this will be helpful for debugging

### How to test your work
Submit a reduction that fails.

Check that the error on the run summary page just says "Error encountered" and there's more details in the actual logs.

### Additional comments
 

Fixes #1228
Fixes #1217



**Before merging ensure the release notes have been updated**
